### PR TITLE
Add a few comments and tweaks

### PR DESCRIPTION
**Pull request by rohanthewiz**
I made a few code improvements. Check out my comments for each. This is how you can come up to speed quickly.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.idea/
```
Comment (rohanthewiz) [inline .gitignore:1]: You always want a `.gitignore` file, so developers have a place to indicate files they want ignored.

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module github.com/jaminyah/GoAjaxForm
```
Comment (rohanthewiz) [inline go.mod:1]: go.mod is the dependency management system recommended for new go projects.
As long as you have a version 1.12 or later version of Go, everything should work as usual transparent to you.

File: main.go [hunk 3]
```diff
@@ -7,11 +7,17 @@ import (
 )
 
 func receiveAjax(w http.ResponseWriter, r *http.Request) {
+	const userErrorMsg = "Sorry, a server error occurred"
```
Comment (rohanthewiz) [inline main.go:10]: Strings that can possibly be reused should be placed in constants. Constants are most always better than loose strings here and there.

File: main.go [hunk 4]
```diff
@@ -7,11 +7,17 @@ import (
...
 	
 	fmt.Println("Received ajax data.")
 
 	// Invoke ParseForm before reading form values
-	r.ParseForm()
+	err := r.ParseForm()
```
Comment (rohanthewiz) [inline main.go:15]: As much as feasible, errors from function calls should be handled or explicitly ignored.

File: main.go [hunk 5]
```diff
@@ -21,17 +27,18 @@ func receiveAjax(w http.ResponseWriter, r *http.Request) {
 	fmt.Printf("Message body: %s\n", r.FormValue("message"))
 
 	// Send back data to client
-	fmt.Fprintf(w, "%s: <span> %s</span><br/>", r.FormValue("username"), r.FormValue("message"))
+	_, _ = fmt.Fprintf(w, "%s: <span> %s</span><br/>", r.FormValue("username"), r.FormValue("message"))
```
Comment (rohanthewiz) [inline main.go:30]: It looks strange, but it is most correct as it is explicitly ignoring the returned values.

File: static/index.html [hunk 6]
```diff
@@ -3,15 +3,19 @@
     <head>
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Ajax form - Go backend</title>
```
Comment (rohanthewiz) [inline static/index.html:6]: Title is required for html pages.

File: static/index.html [hunk 7]
```diff
@@ -3,15 +3,19 @@
...
+       <!-- <script src='//ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js'></script>-->
+        <!-- Yes. For personal projects it is best to pull down
+         the version of jQuery you want directly into your project.
+          Also version 1.n.n of jQuery is way too old -->
+       <script src="scripts/jquery-3.4.1.min.js"></script>
     </head>
     <body>
-        <form action="" method="post" id="comment-form">
-            <label for="name">Your Name:</label>
+        <form action="#" method="post" id="comment-form">
```
Comment (rohanthewiz) [inline static/index.html:15]: Better to use a '#' rather than blank, as it traditionally indicates no action.

File: static/index.html [hunk 8]
```diff
@@ -3,15 +3,19 @@
...
+            <label for="username">Your Name:</label>
```
Comment (rohanthewiz) [inline static/index.html:16]: The `for` of the label should match the `name` and / or `id` of the corresponding `input` or `textarea`
